apps/ui/src/components/TransactionsListItem.vue

```diff
@@ -8,15 +8,27 @@ const props = defineProps<{ tx: Transaction }>();
 
 const title = computed(() => {
   if (props.tx._type === 'sendToken') {
-    return `Send <b>${_n(formatUnits(props.tx._form.amount, props.tx._form.token.decimals))}</b> ${
-      props.tx._form.token.symbol
-    } to <b>_NAME_</b>`;
+    return `Send <b>${_n(
+      formatUnits(props.tx._form.amount, props.tx._form.token.decimals),
+      'standard',
+      {
+        maximumFractionDigits: 2,
```

Comment:
There is no max fractional digit for staking, this limit is just for formatting, using the same formatting as voting power, to avoid long list of `0` for very small number

Reply:
I'm aware of this, but we very often see `~0` in our executions (at least in testing), I wonder if we should give it more space.

Reply:
Reverted to raw value